samples/open-telemetry/application-insights/sample.md

```diff
@@ -58,26 +78,18 @@ snippet: enable-open-telemetry
 
 ### Tracing
 
-The endpoint configures an OpenTelemetry trace provider that includes the `NServiceBus.Core` source and exports traces to Azure Monitor.
+The endpoint configures an OpenTelemetry trace provider that includes the `NServiceBus.Core` source and export traces to Azure Monitor.
 
 snippet: enable-tracing
 
 ### Meters
 
-The endpoint also configures an OpenTelemetry meter provider that includes the `NServiceBus.Core` meter and exports data to Azure Monitor.
+The endpoint also configures an OpenTelemetry meter provider that includes the `NServiceBus.Core` meter and export data to Azure Monitor.
 
 snippet: enable-meters
 
-#### Meter shim
-
-NOTE: The Azure Monitor exporter package does not currently support exporting meter data. The sample includes custom code to collect the meter.
-
-The custom meter exporter captures all meters that begin with `nservicebus.` of the `long` sum type, and forwards them to a `TelemetryClient`.
-
-snippet: custom-meter-exporter
-
-This exporter is installed into the meter provider builder with a custom extension method. The exporter is wrapped by a `PeriodicExportingMetricReader` instance that executes the exporter once every 10 seconds.
+#### Critical time and processing time
 
-snippet: custom-meter-exporter-installation
+[Critical time and processing time captured by the metrics package](/monitoring/metrics/definitions.md#metrics-captured) is not yet included in the built-in open telemetry support so a shim is needed to make sure that the endpoint exposes them as open telemetry metrics.
 
-NOTE: The shim passes `QueueName` as a custom dimension which allows filtering the graphs in Application Insights. Multi-dimensional metrics are not enabled by default. Check [the Azure Monitor documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/app/get-metric#enable-multi-dimensional-metrics) for instructions on how to enable this feature.
```

Comment:
I'm not sure this should be entirely removed. 

From MS docs:
> All metrics that you send using [OpenTelemetry](https://learn.microsoft.com/en-us/azure/azure-monitor/app/opentelemetry-add-modify), [trackMetric](https://learn.microsoft.com/en-us/azure/azure-monitor/app/api-custom-events-metrics#trackmetric) or [GetMetric and TrackValue](https://learn.microsoft.com/en-us/azure/azure-monitor/app/api-custom-events-metrics#getmetric) API calls are automatically stored in both logs and metrics stores. These metrics can be found in the customMetrics table in Application Insights and in Metrics Explorer under the Custom Metric Namespace called "azure.applicationinsights". Although the log-based version of your custom metric always retains all dimensions, the pre-aggregated version of the metric is stored by default with no dimensions. Retaining dimensions of custom metrics is a Preview feature that can be turned on from the [Usage and estimated cost](https://learn.microsoft.com/en-us/azure/azure-monitor/cost-usage#usage-and-estimated-costs) tab by selecting With dimensions under Send custom metrics to Azure Metric Store.

Maybe we should rephrase it and link to this: https://learn.microsoft.com/en-us/azure/azure-monitor/app/pre-aggregated-metrics-log-metrics#custom-metrics-dimensions-and-pre-aggregation

Reply:
Added the note back in